### DISCUSSION TITLE
Things look pretty similar when compared with hyprland so cant there be just plugin which can enable this type of scrolling ?

### DISCUSSION THREAD

**Discussion by Greatz08** [YaLTeR/niri#492] [General]
Just saw sample video and nothing much special i saw so rather than installing and using this i think if possible if plugin could be created for hyprland to have this scrolling kind of feature then it would be much better as Hyprland already is well established project which has grown alot.Just my thoughts so sharing it here

**Comment by YaLTeR** [comment]
A Hyprland plugin could be created, in fact multiple people already did just that:

* https://gitlab.com/magus/hyprslidr
* https://github.com/dawsers/hyprscroller

And you're of course welcome to use those if you'd like. Moreover, you could also use PaperWM which works on top of GNOME Shell (and is a direct inspiration for niri).

I'm here having fun writing my own compositor though!

**Comment by lpnh** [comment]
Hm, it seems like you really enjoy [comparing things](https://github.com/sxyazi/yazi/discussions/806), don't you?

**Comment by TheZoq2** [comment]
For me, a nice community and it being written in Rust are huge advantages, big enough for me to switch over from Hyprland despite having it set up in a way that I'm very happy with

**Reply by mcbeth** [reply]
Agreed.  I came over so I could stop dealing with the community toxicity there, and fell in love with the window management style of niri.  Thanks for the hard work, YaLTeR
